### PR TITLE
Display all news in changeAbout

### PR DESCRIPTION
**Pull request by Rits1272**
## Issue that this pull request solves

Closes: #268 

## Proposed changes
A user will now be able to change his about the section according to the category of the keyword of news he desires.

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

Use : 
- The user starts the script with `python3 -m wplay --wabout`.
- The user is then prompted to `input` the kind of news he wants(for eg. football or coronavirus)
- The about section then gets changed with the kind of news opted by the user. The about section keeps changing as soon as the latest news is available.

Screenshot (59)

### CODE REVIEW

**Comment by Rits1272** [comment]
Please review @xandao6 @rpotter12 @priyansh19

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ wplay/changeAbout.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5278979&bid=17285228)

**Comment by Rits1272** [comment]
Added the screenshot @rpotter12. Please check :)

**Comment by Rits1272** [comment]
How should I proceed @rpotter12? Actually I had asked @xandao6 if I should make any change in the `changeAbout.py`. So he proposed me the changes as done in this PR.

So in this PR, A user can now have news of any `category or keyword` of his liking which will update in every latest news fetch(if any). The category or keyword can be literally anything of the user's choice.

**Comment by priyansh19** [comment]
> How should I proceed @rpotter12? Actually I had asked @xandao6 if I should make any change in the `changeAbout.py`. So he proposed me the changes as done in this PR.
> 
> So in this PR, A user can now have news of any `category or keyword` of his liking which will update in every latest news fetch(if any). The category or keyword can be literally anything of the user's choice.

Yes, you can proceed with an idea like if I enter a keyword to find news e.g "corona" then it will display trending 5 news and from which I can select one of them to send

**Comment by Rits1272** [comment]
@xandao6 have you added the `NEWS_API_KEY` in the `.env` file.You can have api key from [here](https://newsapi.org)

**Comment by xandao6** [comment]
You will need to write a tutorial, otherwise, how can people use this tool?

**Comment by priyansh19** [comment]
@Rits1272 Add a Section for this tool in README.md 
It should contain a few things like :
- How to get your own News API keys
- How to run this program .. (Fully detailed)

Quick addon - If you can add the debugging procedure then you may get extra score for that :v:

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ wplay/changeAbout.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5278979&bid=17285228)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity decreasing per file
==============================
+ wplay/changeAbout.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5278979&bid=17285228)

File: README.md [hunk 1]
```diff
@@ -29,6 +29,13 @@ It is command line software through which you can play with your WhatsApp. It is
 
 ***changeAbout*** is a script to change the about section of your profile. User can select personal quote, news, quotes, etc.
 
+### Steps to use changeAbout
+The changeAbout script uses a news api. For fetching content from the api, you need to generate an api key from [here](https://newsapi.org/). After getting the api key you need to create a `.env` file and then place this key in `.env` file as `NEWS_API_KEY=YOUR KEY`. Now you simple need to run the script:
+
+- `python3 -m wplay --wabout`
+- The script will then prompt the user to input `news's theme`. The news can be related to any **keyword**. For example - Football, Bitcoin etc. You can also specify the category of the news(eg. Business, Sports etc).
+- The script will then change your about section automatically on every latest news fetched from the api related to your input.
```
Comment (rpotter12) [inline README.md:37]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5278979)

File: README.md [hunk 2]
```diff
@@ -29,6 +29,13 @@ It is command line software through which you can play with your WhatsApp. It is
 
 ***changeAbout*** is a script to change the about section of your profile. User can select personal quote, news, quotes, etc.
 
+### Steps to use changeAbout
+The changeAbout script uses a news api. For fetching content from the api, you need to generate an api key from [here](https://newsapi.org/). After getting the api key you need to create a `.env` file and then place this key in `.env` file as `NEWS_API_KEY=YOUR KEY`. Now you simple need to run the script:
+
+- `python3 -m wplay --wabout`
+- The script will then prompt the user to input `news's theme`. The news can be related to any **keyword**. For example - Football, Bitcoin etc. You can also specify the category of the news(eg. Business, Sports etc).
```
Comment (rpotter12) [inline README.md:36]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5278979)

File: README.md [hunk 3]
```diff
@@ -29,6 +29,13 @@ It is command line software through which you can play with your WhatsApp. It is
 
 ***changeAbout*** is a script to change the about section of your profile. User can select personal quote, news, quotes, etc.
 
+### Steps to use changeAbout
+The changeAbout script uses a news api. For fetching content from the api, you need to generate an api key from [here](https://newsapi.org/). After getting the api key you need to create a `.env` file and then place this key in `.env` file as `NEWS_API_KEY=YOUR KEY`. Now you simple need to run the script:
+
+- `python3 -m wplay --wabout`
```
Comment (rpotter12) [inline README.md:35]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5278979)

**Comment by Rits1272** [comment]
I have added the documentation for the `changeAbout` in the `Readme` file. Please review @priyansh19 @rpotter12 @xandao6

**Pull request merged**